### DISCUSSION TITLE
Getting error

### DISCUSSION THREAD

**Discussion by d1pakda5** [projectdiscovery/retryabledns#65] [General]
Hi I am getting below error

github.com/projectdiscovery/retryabledns.(*Client).QueryMultiple(0xc0000fdb80, {0xc0000342f0, 0xc}, {0xc000316340, 0x1, 0x0?})
	/root/go/pkg/mod/github.com/projectdiscovery/retryabledns@v1.0.13/client.go:240 +0xa4a
github.com/projectdiscovery/dnsx/libs/dnsx.(*DNSX).QueryMultiple(...)
	/root/go/pkg/mod/github.com/projectdiscovery/dnsx@v1.1.0/libs/dnsx/dnsx.go:84
github.com/projectdiscovery/dnsx/internal/runner.(*Runner).worker(0xc0002c4140)
	/root/go/pkg/mod/github.com/projectdiscovery/dnsx@v1.1.0/internal/runner/runner.go:555 +0x145
created by github.com/projectdiscovery/dnsx/internal/runner.(*Runner).startWorkers
	/root/go/pkg/mod/github.com/projectdiscovery/dnsx@v1.1.0/internal/runner/runner.go:541 +0xbd

**Comment by ehsandeep** [comment]
@d1pakda5 do share the additional details to replicate this crash for further investigation.
